Automattic/newspack-plugin
# chore(admin): bundle tachyons with WP assets instead of fetching from…

Small tweak – the tachyons CSS library is fetched from unpkg.com. This service might take longer to respond, or be unavailable – it's faster and more reliable to fetch the library from NPM and serve it locally. 

## To test

Go to Analytics wizard -> Custom dimensions, and observe that `flex` and `justify-end` classes are effective and "defined" in the CSS:

<img width="2009" alt="image" src="https://user-images.githubusercontent.com/7383192/206680847-e4bc8856-66d3-44e7-ac1c-6bd5217bb433.png">
